salt/_modules/metalk8s_volumes.py

```diff
@@ -451,6 +451,15 @@ def __init__(self, volume):
             self._kind = DeviceType.PARTITION
         self._kind = DeviceType.DISK
 
+    @property
+    def persistent_path(self):
+        if self._kind == DeviceType.LVM:
+            for symlink in glob.glob('/dev/disk/by-id/dm-uuid-LVM-*'):
+                realpath = os.path.realpath(symlink)
+                if os.path.basename(realpath) == name:
```

Comment:
IIUC, `name` should be computed (maybe make this an attribute or a dynamic property?):

```suggestion
                if os.path.basename(realpath) == device_name(self.path):
```

Reply:
Indeed, `name` is not defined >_<

Fixed.